docs/preview/03-Features/logging-azure-functions.md

```diff
@@ -19,11 +19,13 @@ PM > Install-Package Arcus.WebApi.Logging.AzureFunctions
 The `AzureFunctionsExceptionHandlingMiddleware` class can be added to the Azure Functions worker pipeline to log unhandled exceptions that are thrown during request processing.
```

Comment:
I see it is already mentionned in the title, but should we put an additional 'warning' block here that specifies that this is only applicable for isolated Azure Functions ?

Reply:
Fine by me 👍

Reply:
Worth noting is that this functionality will not be even showing in in-process Azure Functions as those ones don't have middleware, so they will never come in contact with it. But I'll add a warning just in case.

Reply:
> Worth noting is that this functionality will not be even showing in in-process Azure Functions as those ones don't have middleware, so they will never come in contact with it. But I'll add a warning just in case.

I think the warning in the documentation is very helpful, as it is possible that ppl will be wondering 'where is that extension method', while working on a non-isolated project.  The 'warning' might then give them an 'aha erlebnis'.